structuralcodes/materials/reinforcement/__init__.py

```diff
@@ -0,0 +1,72 @@
+"""Reinforcement material."""
+
+import typing as t
+
+from structuralcodes.codes import _use_design_code
+
+from ._reinforcement import Reinforcement
+from ._reinforcementEC2_2004 import ReinforcementEC2_2004
+from ._reinforcementEC2_2023 import ReinforcementEC2_2023
+from ._reinforcementMC2010 import ReinforcementMC2010
+
+__all__ = [
+    'create_reinforcement',
+    'Reinforcement',
+    'ReinforcementMC2010',
+    'ReinforcementEC2_2004',
+    'ReinforcementEC2_2023',
+]
+
+REINFORCEMENTS: t.Dict[str, Reinforcement] = {
+    'fib Model Code 2010': ReinforcementMC2010,
+    'EUROCODE 2 1992-1-1:2004': ReinforcementEC2_2004,
+    'EUROCODE 2 1992-1-1:2023': ReinforcementEC2_2023,
+}
+
+
+def create_reinforcement(
```

Comment:
I express the doubt here, but is valid for the constructors of all reinforcement concrete classes:
fyk and E are not the only characteristics needed.
Important parameters e.g. for sectional design is ultimate strain eps_uk and (optionally if you want to include hardening) the tensile strength ftk.

In many codes (EC, MC, etc.) there are expressed the definitions for ductility classes, does it makes sense this idea:
1) the user can input in the constructor (optionally) the values for ultimate strength and strain. 
Alternatively:
2) The user can input the class letter (A, B, C, D) and we compute ftk and eps_uk from the requirements of each class.

In this way for design the user can select a steel 450C for instance as (fyk = 450, class ='C'). For verification (e.g. evaluation of existing structure) the user may have from experimental tests the values and create the material as (fyk = 450, ftk = 600, eps_uk = 0.12)

What do you think @mortenengen ?

Reply:
Perfect